runtime/shibuya/src/lib.rs

```diff
@@ -1535,6 +1539,70 @@ impl pallet_migrations::Config for Runtime {
     type WeightInfo = pallet_migrations::weights::SubstrateWeight<Runtime>;
 }
 
+parameter_types! {
+    pub const EnterDuration: BlockNumber = 4 * HOURS;
+    pub const EnterDepositAmount: Option<Balance> = None;
+    pub const ExtendDuration: BlockNumber = 2 * HOURS;
+    pub const ExtendDepositAmount: Option<Balance> = None;
+    pub const ReleaseDelay: Option<u32> = None;
+}
+
+/// Calls that can bypass the safe-mode pallet.
+pub struct SafeModeWhitelistedCalls;
+impl Contains<RuntimeCall> for SafeModeWhitelistedCalls {
+    fn contains(call: &RuntimeCall) -> bool {
+        match call {
+            RuntimeCall::Sudo(_)
+            // System and Timestamp are required for block production
+            | RuntimeCall::System(_)
+            | RuntimeCall::Timestamp(_)
```

Comment:
Dumb question - aren't these calls inherent though?
If such calls also need to be filtered, parachain system should also be included here.

Reply:
If I remove `Timestamp` block production stops:

> ❌️ Mandatory inherent extrinsic returned error. Block cannot be produced.  

But, I removed `SafeMode` since the safe-mode pallet cannot disable it's own calls and I added `ParachainSystem`.

Reply:
Hmmm strange, have to admit that I'm surprised :)